modules/cloudwatch/main.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_cloudwatch_log_group" "this" {
+  name              = "devops-ic-${var.env_namespace}-log-group"
```

Comment:
As you can see from Terraform Plan the name will be devops-ic-devops_ic-devops-ic-staging-log-group so I don't think we also need prefix devops-ic-.

I also see a second devops-ic. This seems to also be the value in your environment - ie devops-ic-staging. What about removing the name from environment too? We got it 3 times here 😆

ps why is it named env_namespace here

Reply:
xD, because I set the `environment` value to be `devops-ic-${workspace}` lol, `devops-ic` everywhere.
I updated them all to be more readable in [72c2665](https://github.com/nvminhtue/tyson_devops_ic/pull/26/commits/72c2665759dfea943a523ac814e0a71d53f3e0c6)